docs/content/rest-helix/_index.md

```diff
@@ -69,9 +69,15 @@ Games:
 - [Games -> Search](./games-get)
 - [Games -> Top Games](./games-top)
 
+Moderators:
```

Comment:
Could we call this section `Moderation`? Then we can add endpoints like `getBannedEvents` and `checkAutoModStatus` under this umbrella